src/caf/toolkit/translation.py

```diff
@@ -853,6 +1034,131 @@ def pandas_vector_zone_translation(
     )
 
 
+def pandas_multi_vector_zone_translation(
+    vector: pd.DataFrame,
+    translation: pd.DataFrame,
+    translation_from_col: str,
+    translation_to_col: str,
+    translation_factors_col: str,
+    from_unique_index: list[Any] = [],
+    to_unique_index: list[Any] = [],
+    check_totals: bool = True,
+) -> pd.DataFrame:
+    """Efficiently translate a multi-column pandas vector between index systems.
```

Comment:
Please check this docstring and update accordingly to ensure its accurate. I've just copied and pasted it from the single-vector for now